<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QVTKOpenGLWidget" name="qvtkWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="listView">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </item>
      <item>
       <widget class="QCheckBox" name="BoxClip">
        <property name="text">
         <string>Clip Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="BoxShrink">
        <property name="text">
         <string>Shrink Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Filter">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Reset Cam</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Filter_T">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Filter Test</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openButton"/>
    <addaction name="saveButton"/>
   </widget>
   <widget class="QMenu" name="menuPreloaded">
    <property name="title">
     <string>Preloaded</string>
    </property>
    <addaction name="disk"/>
    <addaction name="arrow"/>
    <addaction name="cone"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Color</string>
    </property>
    <addaction name="ColorChoose"/>
    <addaction name="Color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPreloaded"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openButton"/>
   <addaction name="saveButton"/>
   <addaction name="disk"/>
   <addaction name="arrow"/>
   <addaction name="cone"/>
   <addaction name="ColorChoose"/>
  </widget>
  <action name="openButton">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/file.png</normaloff>:/Icons/file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="saveButton">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/save.png</normaloff>:/Icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="disk">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/disk.png</normaloff>:/Icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Disk Item</string>
   </property>
  </action>
  <action name="arrow">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/arrow.png</normaloff>:/Icons/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Arrow Item</string>
   </property>
  </action>
  <action name="cone">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/slz.png</normaloff>:/Icons/slz.png</iconset>
   </property>
   <property name="text">
    <string>Cone Item</string>
   </property>
  </action>
  <action name="CFilter">
   <property name="text">
    <string>Clip Filter</string>
   </property>
  </action>
  <action name="SFilter">
   <property name="text">
    <string>Shrink Filter</string>
   </property>
  </action>
  <action name="ColorChoose">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/dark.png</normaloff>:/Icons/dark.png</iconset>
   </property>
   <property name="text">
    <string>BaackGround Color</string>
   </property>
  </action>
  <action name="Color">
   <property name="text">
    <string>Actor Color</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
